lib/iris/analysis/__init__.py

```diff
@@ -1499,18 +1499,21 @@ def _weighted_percentile(
         return result
 
 
-@_build_dask_mdtol_function
-def _lazy_count(array, **kwargs):
-    array = iris._lazy_data.as_lazy_data(array)
+def _count(array, **kwargs):
+    """
+    Counts the number of points along the axis that satisfy the condition
+    specified by ``function``.  Uses Dask's support for NEP13/18 to work as
+    either a lazy or a real function.
```

Comment:
Though it occurs to me that this rather depends whether the passed `function` can cope with lazy data.  Probably always did.

Reply:
Yes I agree. It's possibly a bit of a hole, but it's no worse that it was before
:crossed_fingers: